bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableIO.java

```diff
@@ -605,13 +608,22 @@ public AbstractCloudBigtableTableWriteFn(CloudBigtableConfiguration config) {
     @Override
     public void startBundle(Context context) throws Exception {
```

Comment:
Since we are not doing anything in startBundle(), consider removing this method override.